tests/Common/CommonSchemaTest.php

```diff
@@ -405,6 +405,23 @@ public function testGetTableNames(array $pdoAttributes): void
         $db->close();
     }
 
+    public function testHasTable(): void
+    {
+        $db = $this->getConnection(true);
+
+        $schema = $db->getSchema();
+
+        $this->assertTrue($schema->hasTable('customer'));
+        $this->assertTrue($schema->hasTable('category'));
+        $this->assertFalse($schema->hasTable('no_such_table'));
+
+        $this->assertTrue($schema->hasTable('customer', '', true));
+        $this->assertTrue($schema->hasTable('category', '', true));
+        $this->assertFalse($schema->hasTable('no_such_table', '', true));
```

Comment:
It doesn't make sense, because with both values (`refresh` is true or false) result is equal.

Reply:
These validations were requested by @Tigrov

Reply:
> These validations were requested by @Tigrov

Yes, but it was suggested differently

https://github.com/yiisoft/db/pull/913#discussion_r1903200876